### DISCUSSION TITLE
GitHub Copilot is not availabe in VS Extensions - altough I've Visual Studio 17.4

### DISCUSSION THREAD

**Discussion by plisol** [community/community#48486] [Copilot Conversations]
### Select Topic Area

Product Feedback

### Body

I can not install 
GitHub Copilot is not availabe in VS Extensions - altough I've Visual Studio 17.4

![grafik](https://user-images.githubusercontent.com/124688110/221192568-0109db44-1432-41d8-80d0-daad0f4d85fb.png)

![grafik](https://user-images.githubusercontent.com/124688110/221192417-8f9480dd-2eed-47aa-bd04-e3ff49e80426.png)

**Comment by tilmann1** [comment]
@plisol Can you please update to VS 17.5? (or alternatively any VS version >= VS 17.4.4, older versions than VS 17.4.4 will not work with the latest Copilot VS extension.

The other reason why you might not be able see the Copilot VS extension is if you're using Visual Studio for Arm64 (which is currently not supported by Copilot).

Hope this help, thanks!

**Comment by plisol** [comment]
Didn’t solve the problem

Gesendet von Outlook für iOS<https://aka.ms/o0ukef>
________________________________
Von: Tilmann Scheller ***@***.***>
Gesendet: Friday, February 24, 2023 3:54:23 PM
An: community/community ***@***.***>
Cc: plisol ***@***.***>; Mention ***@***.***>
Betreff: Re: [community/community] GitHub Copilot is not availabe in VS Extensions - altough I've Visual Studio 17.4 (Discussion #48486)


@plisol<https://github.com/plisol> Can you please update to VS 17.5? (or alternatively any VS version >= VS 17.4.4, older versions than VS 17.4.4 will not work with the latest Copilot VS extension.

The other reason why you might not be able see the Copilot VS extension is if you're using Visual Studio for Arm64 (which is currently not supported by Copilot).

Hope this help, thanks!

**Reply by tilmann1** [reply]
@plisol Which platform are you on? Are you seeing this with Windows running on a x86_64 host?

**Reply by plisol** [reply]
Windows 11 on 64 host

**Reply by tilmann1** [reply]
@plisol Can you please try to download the extension directly from https://marketplace.visualstudio.com/items?itemName=GitHub.copilotvs (see the green download button at the top) and then double click the extension binary? Thanks!
